NAMESPACE

```diff
@@ -16,6 +16,12 @@ export(indicator_rows)
 export(load_data)
 export(mod_quality_overview_server)
 export(mod_quality_overview_ui)
+export(mod_sidebar_qo_server)
+export(mod_sidebar_qo_ui)
+export(mod_table_legend_server)
+export(mod_table_legend_ui)
+export(mod_top_navbar_server)
+export(mod_top_navbar_ui)
```

Comment:
Is it OK to skip the `mod_` prefix, or do you think we need it?

Reply:
It can be skip:)